### DISCUSSION TITLE
Case à cocher qui se décoche si erreur dans formulaire lors de la validation

### DISCUSSION THREAD

**Discussion by GigMatt** [MTESSDev/FRW#68] [Annonces]
Pouvons-nous avoir une case à cocher obligatoire qui pourrait être désactivée si une erreur existe lors de la validation du formulaire?

Cette case à cocher se nommerait grosso modo: "Je déclare que mes renseignements sont exacts".  
S'il arrive une erreur, il est important de pouvoir revenir faire la correction dans le formulaire., d'où l'importance d'avoir une case "déclaration" décochée. Une fois la correction fait par l'utilisateur, il pourrait cocher à nouveau cette "déclaration".

Est-ce réalisable? 
Merci

**Comment by vinnyboii** [comment]
Bonjour,

Pouvez-vous créer une issue avec ces informations, nous pourrons la prioriser et vous partager nos réflexions quand on y travaillera.

Merci

**Reply by GigMatt** [reply]
Fait #71 

-> Merci
